### PR TITLE
TOAZ-155 AzureTests doesn't clean up created resources after completion

### PR DESCRIPTION
**Pull request by sergiygetlin**
https://broadworkbench.atlassian.net/browse/TOAZ-155

Changes:
Fix azuretests so they clean up created resources after themselves.

### CODE REVIEW

**Review by Qi77Qi — APPROVED** [review]

**Comment by Qi77Qi** [comment]
thanks @sergiygetlin !

File: service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/CreateAndDeleteAzureControlledResourceFlightTest.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package bio.terra.workspace.service.resource.controlled.cloud.azure;
 
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (rtitle) [inline service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/CreateAndDeleteAzureControlledResourceFlightTest.java:3]: I think the WSM style guide says no `*` imports. I have the following in my IDE:
image

**Review by rtitle — APPROVED** [review]
Looks good to me if tests pass - thanks for fixing.

File: service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/CreateAndDeleteAzureControlledResourceFlightTest.java (hunk 1 above)
Comment (sergiygetlin) [reply service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/CreateAndDeleteAzureControlledResourceFlightTest.java:3]: I have another PR which I am currently working. Will fix this problem there.

**Pull request merged**